mypy/stubgenc.py

```diff
@@ -18,6 +18,19 @@
 )
 
 
+# Members of the typing module to consider for importing by default.
+_DEFAULT_TYPING_IMPORTS = (
+    'Any'
```

Comment:
@AWhetter There is a missing comma here, without it you get `'AnyDict'` as the first entry.

Reply:
I've addressed this here: https://github.com/python/mypy/pull/9900